diff --git a/SCI/src/LinearHE/CMakeLists.txt b/SCI/src/LinearHE/CMakeLists.txt
index 37a518e7..2b58827b 100644
--- a/SCI/src/LinearHE/CMakeLists.txt
+++ b/SCI/src/LinearHE/CMakeLists.txt
@@ -13,7 +13,7 @@ if (NOT SEAL_FOUND)
     endif ()
     execute_process(COMMAND ${CMAKE_COMMAND} -DCMAKE_INSTALL_PREFIX=${PROJECT_SOURCE_DIR}/build .
             WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}/extern/SEAL/native/src")
-    execute_process(COMMAND ${CMAKE_COMMAND} --build . --target install
+    execute_process(COMMAND ${CMAKE_COMMAND} --build . --target install --parallel
         WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}/extern/SEAL/native/src")
     find_package(SEAL 3.3.2 EXACT REQUIRED PATHS "${PROJECT_SOURCE_DIR}/build/" NO_DEFAULT_PATH)
 endif()
diff --git a/SCI/src/cleartext_library_float.h b/SCI/src/cleartext_library_float.h
index 37007ee5..e76ae887 100644
--- a/SCI/src/cleartext_library_float.h
+++ b/SCI/src/cleartext_library_float.h
@@ -22,6 +22,7 @@ SOFTWARE.
 #ifndef LIBRARY_CLEARTEXT_FLOAT_H__
 #define LIBRARY_CLEARTEXT_FLOAT_H__
 
+#include <cstdint>
 #include <vector>
 #include <math.h>
 #include <cstdlib>
diff --git a/SCI/src/utils/CMakeLists.txt b/SCI/src/utils/CMakeLists.txt
index 225bfc4f..57d01684 100644
--- a/SCI/src/utils/CMakeLists.txt
+++ b/SCI/src/utils/CMakeLists.txt
@@ -9,7 +9,7 @@ add_library(SCI-utils INTERFACE)
 
 target_compile_options(SCI-utils
     INTERFACE
-    "-pthread;-maes;-msse4.1;-mavx;-mavx2;-faligned-new;-std=c++17;-O3;-Wno-parentheses-equality;-Wno-deprecated-declarations;-Wno-macro-redefined")
+    "-pthread;-maes;-msse4.1;-mavx;-mavx2;-faligned-new;-std=c++20;-O3;-Wno-parentheses-equality;-Wno-deprecated-declarations;-Wno-macro-redefined")
 #     "-pthread;-Wall;-maes;-msse4.1;-mavx;-mavx2;-faligned-new;-std=c++17;-w;-g;-ggdb;-O0")
 
 if(USE_RANDOM_DEVICE)
